src/index.html

```diff
@@ -162,6 +175,48 @@ <h2 class="modal__title">New Post</h2>
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script type="module" src="script/index.js"></script>
+    <!-- Avatar modal -->
+    <div class="modal" id="edit-avatar-modal">
+      <div class="modal__content">
+        <div class="modal__header">
+          <h2 class="modal__title">Edit Avatar</h2>
+          <button type="button" class="modal__close-btn"></button>
+        </div>
+        <form class="modal__form" id="edit-avatar-form" novalidate>
+          <label for="avatar-input" class="modal__label"
```

Comment:
The `for` and `id` of the input must **coincide** to connect them together. Or you don't need `for` if the input is **inside** the `label` (it gets connected automatically)